package.json

```diff
@@ -13,7 +13,10 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
-    "@material-ui/core": "^4.9.4",
+    "@fortawesome/free-brands-svg-icons": "^5.13.0",
```

Comment:
Since Material-UI has a GitHub icon, we should just be able to ditch FA as a whole.

Reply:
Right, that left from my initial attempt to import these SVGs.